wren-ai-service/src/providers/engine/wren.py

```diff
@@ -54,8 +55,10 @@ async def dry_run_sql(
             "source": os.getenv("WREN_IBIS_SOURCE"),
             "manifest": os.getenv("WREN_IBIS_MANIFEST"),
             "connection_info": orjson.loads(
-                os.getenv("WREN_IBIS_CONNECTION_INFO", "{}")
-            ),
+                base64.b64decode(os.getenv("WREN_IBIS_CONNECTION_INFO"))
+            )
+            if os.getenv("WREN_IBIS_CONNECTION_INFO")
+            else {},
```

Comment:
could you modify the example for env.dev file as well if we'd like to encode the connection info

Reply:
have discussed offline